msteams-platform/TOC.yml

```diff
@@ -562,6 +562,9 @@
       - name: Extend bot-based message extension as plugin
         href: messaging-extensions/build-bot-based-plugin.md
         displayName: Copilot, extend message extension, search based message extension, M365, Microsoft 365 Chat, Microsoft Copilot for Microsoft 365, customize Microsoft 365 Copilot
+      - name: Debug a copilot plugin with developer mode
+        href: /microsoft-365-copilot/extensibility/debugging-copilot-plugin
```

Comment:
How about use contextual TOC to include this in Teams docs? So reader doesn't lose context